web/src/features/AppConfig/components/AppConfig.tsx

```diff
@@ -480,6 +508,20 @@ class AppConfig extends Component<Props, State> {
                 newItem.value = oldItem.value;
               }
             }
+            // test data for regex
+            if (["text", "password", "textarea"].includes(newItem.type)) {
+              // TODO: remove- this is used for generating test data locally
+              // newItem.validation = {
+              //   regex: "^.{4,20}$",
+              //   message: "should be between 4 and 20 characters",
+              //   hasError: false,
+              // };
```

Comment:
No BE support for this so i'm leaving my test data generation stuff in for now. This adds a length constraint to every text password and textarea field